recipes/proxy.rb

```diff
@@ -20,6 +20,18 @@
   content '<h1>WORKS!</h1>'
 end
 
+execute 'generate resolver config' do
+  command <<-EOH
+    echo resolver $(awk 'BEGIN{ORS=" "} /nameserver/{print $2}' /etc/resolv.conf | sed "s/ $/;/g") >/etc/nginx/conf.d/resolver.conf
+  EOH
+  notifies :reload, 'service[nginx]'
+end
+
```

Comment:
Maybe... seemed hacky to me, but it works for now ;)
I think a custom ohai fact would probably work better, but I've never had to do them in opsworks and didn't want to spend the time just yet. I'll look into it more.